Dockerfile.ocaml

```
@@ -22,3 +22,5 @@ RUN echo ' \
 RUN cat esy.json
 
 RUN esy
+
+Run rm -rf /tmp/app
```

Seems ok to me. Only thing is that the casing here is inconsistent. I'm also not a docker expert and running `rm` seems a bit unorthodox but the alternative of creating another layer seems like form over function.
```suggestion
RUN rm -rf /tmp/app
```

---

This is a very good catch. I don't think `Run` is even accepted by docker